x-pack/plugins/cases/public/common/translations.ts

```diff
@@ -295,6 +295,12 @@ export const MAX_LENGTH_ERROR = (field: string, length: number) =>
     defaultMessage: 'The length of the {field} is too long. The maximum length is {length}.',
   });
 
+export const MAX_TAGS_ERROR = (length: number) =>
+  i18n.translate('xpack.cases.createCase.maxTagsError', {
+    values: { length },
+    defaultMessage: 'The length of tags array is too long. Array must be of length <= {length}.',
```

Comment:
nit: As this is not an API I think we should make the message bit more friendly. What do you think about `Too many tags. The maximum number of allowed tags is 200` or similar?